### PR TITLE
demo: add select a11y demo

### LINKED ISSUE #6119: Create accessibility demo page for select

### PR DESCRIPTION
**Pull request by crisbeto**
Adds an a11y demo page for `md-select`.

Fixes #6119.

### CODE REVIEW

File: src/demo-app/a11y/select/select-a11y.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    </md-optgroup>
+  </md-select>
+</section>
+
+<section>
+  <h2>Colors</h2>
+
+  <div class="select-a11y-spacer">
+    <md-select placeholder="Primary" color="primary">
+      <md-option value="one">One</md-option>
```
Comment (tinayuangao) [inline src/demo-app/a11y/select/select-a11y.html]: Also use real world examples in colored md-select?

Comment (crisbeto) [inline src/demo-app/a11y/select/select-a11y.html]: Done.

File: src/demo-app/a11y/select/select-a11y.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<section>
+  <h2>Single selection</h2>
+  <p>Select your favorite color</p>
+
+  <md-select placeholder="Colors" [(ngModel)]="selectedColor">
```
Comment (jelbourn) [inline src/demo-app/a11y/select/select-a11y.html]: Does each select need an `aria-label`?

Comment (crisbeto) [inline src/demo-app/a11y/select/select-a11y.html]: The select's `aria-label` defaults to the placeholder.

**Comment by andrewseguin** [comment]
Please rebase

**Pull request merged**